### PR TITLE
docs: add java to develop

### LINKED ISSUE #6812: Feature: add java to develop
**Summary**

add java develop doc

### PR DESCRIPTION
**Pull request by TCeason**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

add java to develop

Fixes #6812

### CODE REVIEW

**Comment by BohuTANG** [comment]
```
[ERROR] Error while parsing Markdown front matter.
```

On your local, please cd `website` and run: `yarn build` will check the error.

**Review by BohuTANG — APPROVED** [review]
:+1:

File: docs/doc/20-develop/03-java.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+### Grants Privileges
+
+Grants `ALL` privileges to the user `user1`:
+```sql
+GRANT ALL on *.* TO user1;
+```
+
+## Java
+
+This guideline show how to connect and query to Databend using JDBC. We will be creating a table named `books` and insert a row, then query it.
```
Comment (soyeric128) [inline docs/doc/20-develop/03-java.md:35]: This topic shows how to connect and query Databend using JDBC. We will create a table named `books`,  insert a row, and then query data from the table.

File: docs/doc/20-develop/03-java.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        }
+    }
+}
+```
+
+### Run demo.java
+
+In this case:
+
+The demo classpath is in the folder: /home/eason/database/source-code/test/out/test/test
```
Comment (soyeric128) [inline docs/doc/20-develop/03-java.md:116]: The demo classpath is located at: /home/eason/database/source-code/test/out/test/test

**Pull request merged**